EXOSIMS/Prototypes/Observatory.py

```diff
@@ -553,6 +553,7 @@ def generate_koMap(self,TL,missionStart,missionFinishAbs,koangles):
         extstr += '%s: ' % 'koangles'         + str(koangles) + ' '
         extstr += '%s: ' % 'Name' + str(getattr(TL, 'Name')) + ' '
         extstr += '%s: ' % 'nStars' + str(getattr(TL, 'nStars')) + ' '
+        extstr += '%s: ' % 'StarCatalog' + str(getattr(TL, 'StarCatalog')) + ' '

```

Comment:
I've verified that this change has no effect other than making all existing caches stale.  Please roll it back.